lib/jortt/client/customers.rb

```diff
@@ -69,8 +69,7 @@ def update(id, payload)
       #   client.customers.delete("9afcd96e-caf8-40a1-96c9-1af16d0bc804")
       #
       def delete(id)
-        response = token.delete("/customers/#{id}")
-        response.status == 204
+        super("/customers/#{id}")
```

Comment:
I'm not a fan of inheritance for many different reasons, mainly reduced maintainability, flexibility and understandability of code.

I would suggest replacing it with composition/delegation here. You could create a `Resource` class that is instantiated in resource implementations and use those.